engine-standalone-storage/src/sync/types.rs

```diff
@@ -493,6 +496,7 @@ enum BorshableTransactionKind<'a> {
     StorageUnregister(Option<bool>),
     StorageWithdraw(Cow<'a, parameters::StorageWithdrawCallArgs>),
     SetPausedFlags(Cow<'a, parameters::PauseEthConnectorCallArgs>),
+    SetOwner(Cow<'a, parameters::SetOwnerArgs>),
```

Comment:
This new variant needs to be put at the bottom of the enum (after `Unknown`). The reason is because the borsh serialization is sensitive to the order of the variants as listed in the source code. By adding a new variant in the middle you will shift the variant selection byte in the serialization causing it to fail to deserialize old data.

The order of variants in `TransactionKind` do not matter because it does not implement borsh serialization, but it is very important to put new transaction kinds at the bottom of `BorshableTransactionKind`.